client/src/App.js

```diff
@@ -1,17 +1,26 @@
+//Import is to bring information from the particular address into this file. This file is then able to export information.
 import React from "react";
-import "./App.css";
+import "./App.css"; //Importing the styles from the App.css file. The typical syntax is not used but the address of the CSS file.
 import Button from "./Button";
 import LinearGradient from "./LinearGradient";
 import TextInput from "./TextInput";
 
+//A functional component is the JavaScript function. App is the name of the function.
 function App() {
+  //A syntax of useState hook is Destructuring Assignment what looks like properties of an array. It is concise and readable.
+  /* The useState is a named export from the React library. This hook returns an array of two values - the current state value
+     & the state setter, where we update the value of this state.  Null is an initial state argument, and this indicates 
+     we  don't have the value needed during the first render. */
```

Comment:
Fantastic overview here. I think that explaining what the `null` property does will be particularly useful